sample-error-response-v2.yaml

```diff
@@ -0,0 +1,72 @@
+openapi: '3.0.0'
+info:
+  description: This specification is only for sample error response for API v2.
+  version: '1.0'
+  title: Sample Error Response
```

Comment:
Question: Should we make it even more obvious that this is only for the public API V2, by changing the title to, e.g. `Public API V2 Sample Error Responses`